low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -110,124 +167,132 @@ def make_sspl_support_bundle_tarfile(parser, tmpFile):
             rmq_file.write("======================================================= \n")
         rmq_file.close()
 
-        if sspl_state == "active" and "-noencl" not in parser:
-            url = singleton_realstorencl.build_url("/")
-            COLLECTING_DEBUG_LOG_STARTED = False
-            for encl_trigger_log_retry_index in range(0, ENCL_TRIGGER_LOG_MAX_RETRY):
-                response = singleton_realstorencl.ws_request(url, singleton_realstorencl.ws.HTTP_POST, \
-                    post_data=f'/api/collectDebugData/"{SUPPORT_REQUESTOR_NAME}"{SUPPORT_EMAIL_ID}"{SUPPORT_CONTACT_NUMBER}')
+    def get_consul_data(self):
+        consul_sspl_cfg_cmd = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/bin/consuldump.py -p sspl_config -l /tmp/ sspl/"
+        consul_sspl_cache_cmd = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/bin/consuldump.py -p sspl_cache -l /tmp/ var/{PRODUCT_FAMILY}/sspl"
+        subout = self._run_command(consul_sspl_cfg_cmd)
+        subout = self._run_command(consul_sspl_cache_cmd)
+        del subout
+        # waiting to 2 to 3 second to genrate consul dump file.
+        time.sleep(3)
+        # Get latest consul sspl data from tmp directory.
+        all_subdirs = ['/tmp/'+d for d in os.listdir("/tmp/") if os.path.isdir('/tmp/'+d)]
```

Comment:
cant we just create filenames first & dump data in those directly. figuring out latest generated file can run into trouble later

Reply:
Done